lib/fictium/exporters/api_blueprint_exporter/action_formatter.rb

```diff
@@ -0,0 +1,29 @@
+module Fictium
+  class ApiBlueprintExporter
+    class ActionFormatter
+      def format(action)
+        [build_header(action).strip, build_examples(action)].select(&:present?).join("\n\n")
+      end
+
+      private
```

Comment:
You don't need `#build_footer` here?

Reply:
No footer here, not all exporters do the same.